schemas/com.mailgun/message_clicked/jsonschema/1-0-0

```diff
@@ -0,0 +1,122 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Mailgun message clicked event",
+  "self": {
+    "vendor": "com.mailgun",
+    "name": "message_clicked",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "body-plain": {
+      "description": "text/plain body of the message.",
+      "type": "string",
+      "maxLength": 524288
+    },
+    "campaign-id": {
+      "description": "id of campaign triggering the event.",
+      "type": "string"
+    },
+    "campaign-name": {
+      "description": "The name of campaign triggering the event.",
+      "type": "string"
+    },
+    "city": {
+      "description": "Name of the city the event came from or ‘unknown’ if it couldn’t be determined.",
+      "type": "string",
+      "maxLength": 64
+    },
+    "client-name": {
+      "description": "Name of the client software, e.g. ‘Thunderbird’, ‘Chrome’, ‘Firefox’.",
+      "type": "string",
+      "maxLength": 64
+    },
+    "client-os": {
+      "description": "OS family running the client software, e.g. ‘Linux’, ‘Windows’, ‘OSX’.",
+      "type": "string",
+      "maxLength": 64
+    },
+    "client-type": {
+      "description": "Type of software the link was opened in, e.g. ‘browser’, ‘mobile browser’, ‘email client’.",
+      "type": "string",
+      "maxLength": 64
+    },
+    "country": {
+      "description": "Two-letter country code (as specified by ISO3166) the event came from or ‘unknown’ if it couldn’t be determined.",
+      "type": "string",
+      "maxLength": 2
+    },
+    "device-type": {
+      "description": "Device type the link was clicked on. Can be ‘desktop’, ‘mobile’, ‘tablet’, ‘other’ or ‘unknown’.",
```

Comment:
It looks like complete list. We can use `enum` then.

Reply:
It may be ok to use an enum in this particular case as the data is unlikely to add new values soon (such as 'phablet') but I would like to understand all the uses of the data first. As far as I can tell this will make no difference to Redshift.